### ISSUE TITLE
screen reader user cannot access any dropdown menu except Import

### ISSUE THREAD

**Issue by mehgcap** [rmcrackan/Libation#1866]
PLEASE FILL OUT THE FOLLOWING. Bug reports with limited information or lacking an attached log file may get limited or delayed help.

___

## Describe the bug  
Pressing alt in Libation should jump to the dropdown menus (file, edit, help, import, etc). I can only get to the import menu. This means I can't get to settings, among plenty of other missing options.

## To Reproduce  

1. Open Libation on Windows.
2. Using a screen reader, press the alt key. Narrator will work fine, so press ctrl-win-enter to get easy access to a screen reader. I don't know if this problem happens without one running, though I suspect it does.
3. You are on the Import menu. Use left or right arrow to try to access other menus. Nothing happens.

## Expected behavior  
Alt should jump to the menu bar, letting me access all the menus. Once I press alt, or alt with one of the letters that gets me to a menu, I should be able to left/right arrow between the menus. Alt by itself should land me on the first menu without opening it. I should, in short, be able to get to all the menus with the keyboard.

## Platform  
Windows 11, NVDA 2026.1 (but can replicate with Narrator)

## Log Files
[Log202606_001.log](https://github.com/user-attachments/files/28855686/Log202606_001.log)

**Comment by rmcrackan** [comment]
Thank you so much for your log and for creating this ticket. I pride myself on Libation Classic being as accessible as possible.

Please try these and let me know the result. These tests will let me hone in on exactly what needs fixing:

* `Alt+S` should open Settings directly (mnemonic on &Settings)
* `Alt+L`, `Alt+X`, `Alt+F`, `Alt+V` : other top-level menus without using arrows
* Wait until "Scanning..." disappears from the menu bar, then retry arrows

**Comment by mehgcap** [comment]
Alt with any other letter I tried goes to the import menu. When it does, NVDA confirms the situation by saying "import collapsed 1 of 1". This indicates it sees only one menu, though I've used apps where all menus are present and each reports "1 of 1".

I don't know for sure if "scanning" was still present. However, I left Libation open all night, so I can't imagine it was still scanning.

I should also add that this has been a problem for a while... Maybe a year or more? I'm not sure. I got so used to using libationcli that I almost never use the UI anymore. I wanted to have Libation re-scan for higher quality downloads and ran into this problem again, so I wanted to finally report it properly.

**Comment by rmcrackan** [comment]
I installed NVDA. Using the latest Libation Classic, I get something like 'import collapsed alt plus 1 of 6'. (It's really difficult for me to understand this voice so that may not be exactly correct). I've never used NVDA though so I don't know if I'm reproducing your issue correctly.

**Comment by mehgcap** [comment]
For some reason, what you said made me think of something. Turns out this problem goes away if I maximize the window (alt-space, x). Screen readers don't care if the window is tiny, they only care about what text and controls they can find. I've made webpages before that worked beautifully for me, only to have my boss tell me that all my elements were (visually) on top of each other.

In Libation's case, it seems Windows hides the menus that won't fit on the screen, thus hiding them from NVDA and Narrator as well. Mystery solved, I guess. I'll try to remember to maximize the window if I run into this again. Sorry for the false alarm.

**Comment by rmcrackan** [comment]
No worries. Is there anything I can do on my side to help?

**Comment by mehgcap** [comment]
All I can think of is to open the app always maximized, but I don't know what that might do visually. If there's a way to tell the app to tell Windows to not hide things, that would also work, but I doubt such a thing is possible.

**Comment by rmcrackan** [comment]
It should restore to whatever size you closed it at, including maximized. Are you able to: open, maximize, close, reopen, and see if it's still maximized?

**Comment by mehgcap** [comment]
Yes, that works. This is far from the first time that an application has resized itself without me noticing. Unless odd things like missing content or very few words per line start happening, I never know how big things are on the screen. Libation may stay maximized for months, then randomly become smaller and stay that way until I manually maximize it again. But it does seem to stay at the size it last was, to answer the question. It's just that I don't trust Windows to let me keep it that size.
